src/main/java/de/tum/in/www1/artemis/service/ExamService.java

```diff
@@ -1046,4 +1043,25 @@ public void unregisterStudentFromExam(Long examId, boolean withParticipationsAnd
         log.info("User " + currentUser.getLogin() + " has removed user " + student.getLogin() + " from the exam " + exam.getTitle() + " with id " + exam.getId()
                 + ". This also deleted a potentially existing student exam with all its participations and submissions.");
     }
+
+    /**
+     * Adds all students registered in the course to the given exam
+     *
+     * @param courseId Id of the course
+     * @param examId Id of the exam
+     * @return the list of students who could not be registered for the exam, because they could NOT be found in the Artemis database and could NOT be found in the TUM LDAP
+     */
+    public List<StudentDTO> addAllStudentsOfCourseToExam(Long courseId, Long examId) {
+        Course course = courseService.findOne(courseId);
+        var users = userService.findAllUsersInGroup(course.getStudentGroupName());
```

Comment:
I think you can directly assign those users to the exam

Reply:
Looks better now. I suggest you use the method `getStudents(course)` here instead, which would be easier ;-)

<img width="450" alt="image" src="https://user-images.githubusercontent.com/744067/106378960-e904e600-63a8-11eb-8016-6df47f29d22d.png">

Reply:
Done!